Changelog.md

```diff
@@ -1,5 +1,7 @@
 ### New rules
 * A tabpanel should be related to a tab via aria-controls or aria-labelledby (`src/audits/UncontrolledTabpanel.js)
+* A data table must identify row and column headers (`src/audits/TableHasAppropriateHeaders.js`)
+* A tabpanel should be related to a tab via aria-controls or aria-labelledby (`src/audits/UncontrolledTabpanel.js)
```

Comment:
The backtick is missing, but also looks like some new rules got duplicated here in the changelog merge. I've fixed both. :)